### DISCUSSION TITLE
버튼 스타일 컴포넌트 추가/수정 제안

### DISCUSSION THREAD

**Discussion by kyw0716** [woowacourse-teams/2023-car-ffeine#211] [제안]
현재 버튼 컴포넌트에는 color props가 존재하지 않아 동적으로 버튼 내 글씨 색상을 변경하는 것이 어렵습니다.

color props를 추가해 동적인 색상 변경이 가능해졌으면 좋겠습니다.

**Comment by feb-dain** [comment]
텍스트 스타일 때문에 불필요한 태그를 많이 사용하게 되는 문제
=> 버튼에서 텍스트 스타일까지 줄 수 있도록 수정해야 할 것 같다
